concordium-std/src/traits.rs

```diff
@@ -321,6 +340,163 @@ pub trait DeserialCtx: Sized {
 
 pub trait DeserialStateCtx<S>: Sized
 where
-    S: HasContractStateLL, {
+    S: HasContractStateLL,
+{
     fn deserial_state_ctx<R: Read>(state: &Rc<RefCell<S>>, source: &mut R) -> ParseResult<Self>;
 }
+
+/// A trait for the contract state / host functions API.
+/// It provides abstractions to access the state of the contract, and to invoke host functions.
+pub trait ContractStateApi: Debug {
```

Comment:
What is the point of this trait? It is just adding a trait that names the functions that are imported. What problem does this solve?

Reply:
Since we did not need it until now, I do not think it is needed now for contract state changes either.